app/components/ShareButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Flex, Icon } from '@webkom/lego-bricks';
+import { ShareIcon } from 'lucide-react';
+import { addToast } from 'app/reducers/toasts';
+import { useAppDispatch } from 'app/store/hooks';
+
+type Props = {
+  title: string;
+  url: string;
+};
+
+const ShareButton: React.FC<Props> = ({ title, url }) => {
+  const dispatch = useAppDispatch();
+  const handleShare = async () => {
+    if (navigator.share) {
```

Comment:
```suggestion
    if (navigator?.canShare?.({ title, url })) {
```
The [`canShare`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/canShare) function can be used to check if the browser supports sharing the given data. Also I think optional chaining (`?.`) might be a good idea because some browsers don't support `navigator.share` at all (so it might be `undefined`)